File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/FriendsWithCountersResponse.java
package dev.ragnarok.fenrir.api.model.response;

import com.google.gson.annotations.SerializedName;

import dev.ragnarok.fenrir.api.model.Items;
import dev.ragnarok.fenrir.api.model.VKApiUser;

public class FriendsWithCountersResponse {

    @SerializedName("friends")
    public Items<VKApiUser> friends;

    @SerializedName("counters")
    public VKApiUser.Counters counters;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/FriendsWithCountersResponse.kt
package dev.ragnarok.fenrir.api.model.response

import dev.ragnarok.fenrir.api.model.Items
import dev.ragnarok.fenrir.api.model.VKApiUser
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class FriendsWithCountersResponse {
    @SerialName("friends")
    var friends: Items<VKApiUser>? = null

    @SerialName("counters")
    var counters: VKApiUser.Counters? = null
}